code/modules/fish/fishtank.dm

```diff
@@ -313,14 +313,17 @@
 	egg_list.Cut()									//Destroy any excess eggs, clearing the egg_list
 
 /obj/machinery/fishtank/proc/harvest_fish(var/mob/user)
-	if(!fish_count)									//Can't catch non-existant fish!
+	if(fish_count <= 0)									//Can't catch non-existant fish!	
```

Comment:
fish_count shouldn't be able to reach negative numbers, so this change shouldn't make a difference (because 0 will make `if(!fish_count)` return a true condition)

If they somehow get negative fish in the bowl, then that's a whole new issue in itself that will likely cause runtimes or weird behavior elsewhere in the fish tank code.

Reply:
I actually suggested using `<=` here, as IMO it's better to catch this and deal with it before it becomes a problem then to wait for it to happen and then have it screw something up.

Reply:
For logical conditions like this it doesn't really matter which form you use